modules/sdk-core/src/account-lib/mpc/tss/eddsa/eddsa.ts

```diff
@@ -124,6 +125,21 @@ export default class Eddsa {
     const chaincodes = [uShare, ...yShares].map(({ chaincode }) => bigIntFromBufferBE(Buffer.from(chaincode, 'hex')));
     const chaincode = chaincodes.reduce((acc, chaincode) => (acc + chaincode) % base);
 
+    // Verify shares.
+    for (const share of yShares) {
+      if ('v' in share) {
```

Comment:
do we expect cases where `v` can exist for some yShares but not all? 

also, should we just enforce that all yShares should have `v` or non do?

Reply:
The user key shares in wallets that are currently in production will not have the `v` value. This will be an issue for the HSM, but we have a solution in mind to address this. But we need all this VSS stuff live in the SDK before we can apply it.